teps/0135-coscheduling-pipelinerun-pods.md

```diff
@@ -393,8 +462,8 @@ E2E tests should be added, covering PipelineRuns where multiple PVC workspaces a
 
 As mentioned in ["Configuration"](#configuration), we'll need to replace the "disable-affinity-assistant" flag with the "coscheduling" flag. The steps are as follows:
 
-1. Coscheduling introduced as an alpha feature. Options for the "coscheduling" flag are "disabled" (default), "coschedule-pipelineruns", and "isolate-pipelineruns". The "disable-affinity-assistant" flag is announced as deprecated. If "disable-affinity-assistant" is set to "false", "coscheduling" must be set to "disabled", since the options for coscheduling are incompatible with the existing affinity assistant.
-1. After 9 months, or as part of a v1.0 software release (whichever happens first), the "disable-affinity-assistant" flag will be removed and replaced with a new "coscheduling" option "coschedule-workspaces" in the same release. The "coscheduling" flag will be set to "disabled" by default.
+1. Coscheduling introduced as an alpha feature. Options for the "coscheduling" flag are "coschedule-workspaces" (default), "disabled", "coschedule-pipelineruns", and "isolate-pipelineruns". The "disable-affinity-assistant" flag is announced as deprecated. If "disable-affinity-assistant" is set to "false", "coscheduling" must be set to "coschedule-workspaces", since the options for coscheduling are incompatible with the existing affinity assistant.
+2. After 9 months, or as part of a v1.0 software release (whichever happens first), the "disable-affinity-assistant" flag will be removed and the affinity assistant behavior will be only determined by the "coscheduling" feature flags.
```

Comment:
I think it's worth noting that we may want to consider making "coschedule-pipelineruns" the default value in the future.

Reply:
Updated!